### ISSUE TITLE
Are dependencies in setup.py up to date?

### ISSUE THREAD

**Issue by vastsoun** [google-research/dreamer#8]
Hi,

I tried installing and running this repo within a fresh virtualenv (i.e. no packages other than pip and setuptools installed) via the command:
`pip install -e ~/git/dreamer`.

However, I get an error when running the following command:
`python3 -m dreamer.scripts.train --logdir ./logdir/atari   --params '{defaults: [dreamer, pcont, discrete, atari], tasks: [atari_boxing]}'`

It complains that `AttributeError: 'Categorical' object has no attribute 'probs_parameter'`.

It would seem that the `probs_parameter` method was added in TFP `r0.8`, but `dreamer/setup.py` installs `tensorflow-gpu==1.13.1` and `tensorflow-probability==0.6`.

I resorted to installing `tensorflow-gpu==1.15.2` and `tensorflow-probability==0.8` and it seems to run (I now get `CUDA_ERROR_OUT_OF_MEMORY: out of memory` but that may due to the GPU on my Lenovo P50), but I have no way of telling if this will work as intended. 

Can you please confirm the versions of TF and TFP known to work well with `dreamer`?

Sincerely,
Vassilios

**Comment by Jiankai-Sun** [comment]
Same error for Atari environment. @danijar

**Comment by danijar** [comment]
You're right, the newer TFP library is correct. Also check out the newer code base at http://github.com/danijar/dreamer.
